shub/image/list.py

```diff
@@ -140,12 +143,17 @@ def raise_shub_image_info_error(error):
     if not isinstance(spiders_list, list):
         raise_shub_image_info_error('spiders section must be a list')
 
+    project_type = metadata.get('project_type')
```

Comment:
This allows breaking the contract by not supplying a project type. Will we allow `None` on the server side? Otherwise maybe we should default to `scrapy` or raise an error here (as we declared `project_type` to be required [in #280](https://github.com/scrapinghub/shub/pull/280/files#diff-eac378ae7fa584fd97b26cf7f2799382R36))